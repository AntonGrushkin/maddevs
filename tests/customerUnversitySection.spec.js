import {
  mount
} from '@vue/test-utils';
import CustomerUniversitySection from '../client/components/Blog/CustomerUniversitySection';

describe('Customer University section component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(CustomerUniversitySection, {
      stubs: ['prismic-image'],
      mocks: {
        $prismic: {
          asText: smth => ''
        }
      }
    });
  });

  test('Is a Vue instance', () => {
    expect(wrapper.exists()).toBeTruthy();
  });

  test('Renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
