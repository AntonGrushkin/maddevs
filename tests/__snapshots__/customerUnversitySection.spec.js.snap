// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customer University section component Renders correctly 1`] = `
<section
  class="customer-university"
>
  <div
    class="container"
  >
    <div
      class="customer-university__title"
    >
      
      Customer
      <br />
       
      <span>
        University
      </span>
    </div>
     
    <div
      class="d-flex justify-content-between"
    >
      <div
        class="customer-university__featured-post w-50"
      >
        <!---->
      </div>
       
      <div
        class="w-50"
      >
        <div
          class="customer-university__list-wrapper pl-2"
        >
          <h6
            class="customer-university__list-title"
          >
            Series of articles:
          </h6>
           
          <div />
           
          <div
            class="customer-university__list-show-more-wrapper"
          >
            <button
              class="customer-university__list-show-more"
            >
              
              Browse all series
            
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
